contracts/tokenomics/maker/tests/integration.rs

```diff
@@ -486,15 +487,28 @@ fn collect_all() {
             token_asset(astro_token_instance.clone(), Uint128::from(100_000_u128)),
         ],
     ] {
-        let pair_info = create_pair(
+        create_pair(
             &mut router,
             owner.clone(),
             user.clone(),
             &factory_instance,
             t,
         );
+    }
 
-        pair_addresses.push(pair_info.contract_addr);
+    // set asset to swap
+    for t in vec![
+        native_asset(uusd_asset.clone(), Uint128::zero()),
+        token_asset(astro_token_instance.clone(), Uint128::zero()),
+        native_asset(uluna_asset.clone(), Uint128::zero()),
+        token_asset(usdc_token_instance.clone(), Uint128::zero()),
+        token_asset(test_token_instance.clone(), Uint128::zero()),
+        token_asset(bridge2_token_instance.clone(), Uint128::zero()),
+    ] {
+        assets.push(AssetWithLimit {
```

Comment:
Would be better just to initialize directly
let assets = vec![
 native_asset(uusd_asset.clone(), Uint128::zero()),
        token_asset(astro_token_instance.clone(), Uint128::zero()),
        native_asset(uluna_asset.clone(), Uint128::zero()),
        token_asset(usdc_token_instance.clone(), Uint128::zero()),
        token_asset(test_token_instance.clone(), Uint128::zero()),
        token_asset(bridge2_token_instance.clone(), Uint128::zero()),
    ]